examples/asr_training/rnnt_lightning.py

```diff
@@ -0,0 +1,535 @@
+from argparse import ArgumentParser
```

Comment:
The file name could be improved. for example, `examples/asr/librispeech_emformer_rnnt/train.py`? Suppose we have more recipes in the future like `speech_commands_conformer_ctc/train.py`. What do you think?

Reply:
yeah sounds good — i chose the name rather arbitrarily